source/eprosima-suite/eprosima_suite.rst

```diff
@@ -0,0 +1,191 @@
+:orphan:
```

Comment:
I think this page can be a subsection of "eProsima Suite". That way, we can see it in the TOC tree

Reply:
This :orphan: directive was meant to prevent warning messages since with the current layout there is no TOC tree containing this page. Will try reorganizing it with an intro/contents structure more similar to the other documentation repositories.